### DISCUSSION TITLE
store vs signal

### DISCUSSION THREAD

**Discussion by xobjects** [solidjs/solid#1090] [Q&A]
Consider an app with multiple lists the use a Store for each list.

Actions can be triggered when a list item is selected.  In the past I have used Observables, but I would like to stick with solidjs methodologies for now, assuming it's the right choice.

So, for each list, I need a current_selected_item.  Would it be best to use a Store or a Signal for this?  I realize that for more complex updates, especially on nested objects, that a Store is the way to go.  For a simple object like current_selected_item, is there any compelling reason to use a Store?

**Comment by orenelbaum** [comment]
I assume current_selected_item is an id. So no need for a store. Stores are only meant for nested state when you want nested updates to be applied granularly.
